pkg/writer/junit_xml_test.go

```diff
@@ -85,14 +85,15 @@ func TestJUnitXMLWriter(t *testing.T) {
 				policy.EngineOutput{
 					ViolationStore: &results.ViolationStore{
 						Summary: results.ScanSummary{
-							ResourcePath:     "test_resource_path",
-							IacType:          "k8s",
-							Timestamp:        "2020-12-12 11:21:29.902796 +0000 UTC",
-							TotalPolicies:    550,
-							LowCount:         0,
-							MediumCount:      0,
-							HighCount:        1,
-							ViolatedPolicies: 1,
+							ResourcePath:    "test_resource_path",
+							IacType:         "k8s",
+							Timestamp:       "2020-12-12 11:21:29.902796 +0000 UTC",
```

Comment:
If you could not hardcode the date string, that'd be great and holistic. its fine this way for now as well